src/Illuminate/Routing/Router.php

```diff
@@ -122,7 +122,7 @@ class Router implements RegistrarContract, BindingRegistrar
      * Create a new Router instance.
      *
      * @param  \Illuminate\Contracts\Events\Dispatcher  $events
-     * @param  \Illuminate\Container\Container  $container
+     * @param  \Illuminate\Container\Container|null  $container
```

Comment:
Stuff like this is not correct. The default value if none is provided for this parameter is indeed `null`, but you cannot actually provide `null` as the value of the parameter, otherwise the typehint would have to be `?Container`.

Reply:
@X-Coder264 sure you can: https://3v4l.org/MRT12

Reply:
@driesvints Ok, weird, I was wrong, I would've bet everything that this will/would and has failed for me in the past, but oh well, this is a good TIL :)

Reply:
No problem!